### ISSUE TITLE
Jump links don't match implementation

### ISSUE THREAD

**Issue by ajacobs21e** [patternfly/patternfly-design-kit#389]
- Change section label to sentence case
- add border to subsections
- add mobile label
- add mobile option to template

@mmenestr

**Comment by ajacobs21e** [comment]
[jumplinks.sketch.zip](https://github.com/patternfly/patternfly-design-kit/files/6509296/jumplinks.sketch.zip)
@doruskova is it easier for you if I just update the sketch library myself? once the library is updated I'll do the template updates.

**Comment by doruskova** [comment]
@ajacobs21e Yes, go ahead. Thank you!

**Comment by doruskova** [comment]
@ajacobs21e I have a question :D Do you think that it's necessary to have First, Middle and last items? I think that the only difference is in padding and the padding can be explained in a template file (that's why we do spec). It'd make the structure in a symbol library more clear. WDYT?

**Comment by ajacobs21e** [comment]
@doruskova what about like this?
[jumplinks 2.sketch.zip](https://github.com/patternfly/patternfly-design-kit/files/6532475/jumplinks.2.sketch.zip)
The difference is the padding + the leading or trailing line so I made a separate component that can be put before/after the jump links.

**Comment by doruskova** [comment]
@ajacobs21e Merged!

**Comment by doruskova** [comment]
@ajacobs21e I updated a template file with new symbols. Let me know if it's not correct :) My understanding is that before and after is a 16px long line.

**Comment by ajacobs21e** [comment]
[PatternFly template.sketch.zip](https://github.com/patternfly/patternfly-design-kit/files/6576695/PatternFly.template.sketch.zip)
I added in mobile and subsections. I also updated the existing templates to use separate leading and trailing lines. The symbol has a set size - should we lock it? If you use one symbol that spans the whole width / length of the jump links symbols, then it's annoying to select individual components.
